autogpt_platform/frontend/src/lib/utils.ts

```diff
@@ -397,6 +391,14 @@ export function isEmptyOrWhitespace(str: string | undefined | null): boolean {
   return !str || str.trim().length === 0;
 }
 
+export function isEmpty(value: any): boolean {
+  return (
+    value === undefined ||
+    value === "" ||
+    (typeof value === "object" && _isEmpty(value))
+  );
+}
```

Comment:
I found out the hard way that `lodash.isEmpty(1337)` returns `true`, because `1337` is not a non-empty string nor a non-empty object. This `isEmpty` implementation wraps `lodash.isEmpty` and does handle numbers correctly.